sklearn/frozen/tests/test_frozen.py

```diff
@@ -221,3 +221,16 @@ def test_frozen_params():
     other_est = LocalOutlierFactor()
     frozen.set_params(estimator=other_est)
     assert frozen.get_params() == {"estimator": other_est}
+
+
+def test_frozen_ignores_sample_weight(regression_dataset):
+    X, y = regression_dataset
+    estimator = LinearRegression().fit(X, y)
+    frozen = FrozenEstimator(estimator)
+
+    # Should not raise: sample_weight are just ignored as they are not used.
+    frozen.fit(X, y, sample_weight=np.ones(len(y)))
+
+    # FrozenEstimator should not have sample_weight in its signature to make it
+    # explicit that sample_weight are accepted and ignored intentionally.
+    assert has_fit_parameter(frozen, "sample_weight")
```

Comment:
Isn't the comment conflicting what we are actually testing? It seems like sample_weight is now part of the signature of the fit method.